tutorial/go-live-checklist.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Checklist
+parent: Go-Live Checklist
+grand_parent: Malibu Tutorial
+nav_order: 01
+---
+
+# {{page.title}}
+
+*This tutorial was contributed by [Deo Kumar](https://www.linkedin.com/in/deo-kumar)*
+
+There are few points that needs to be check before and after Go-Live
+
+## Before Go-live checklist
+
+### BlackKnight config
+
+To make sure if the config in blackknight is set properly. If not then follow [Black Knight Tutorial]({{"/tutorial/deploying-with-black-knight" | absolute_url}}) to setup you initial config. Once the initial setup is done, you need to add your respective feature configurations in the config file of blackknight.
+
+Example(Malibu): Publisher.yml -> `enableLogin: true`
+
+![Publisher config]({{"images/publisher-config.png" | absolute_url}})
+
+More about blackknight, refer docs [here]({{"/tutorial/deploying-with-black-knight" | absolute_url}})
+
+### Production API credentials for third-party applications
+
+Verify all the third-party credentials are whitelisted and added to respective places, like Blackknight config, Bold, Bridgekeeper database, and Accesstype dashboard.
+
+Example:
+- GA/GTM ID
+- Metype ID and Whitelist the domain.
+- Push notifications keys
+- Social login
+  - Google client id and secret id
+  - Facebook app id and secret id
+  - Apple app id's, service id and keys
+
+Apart from the above points below important points also need to be verified
+
+- Structured data. Verify structured data in homepage(Organisation, Website) stories. (Article, Organisation) (News Article Schema)
+- Meta description
+- Amp pages
+- Shrubbery analytics
+- Instant article feed.
```

Comment:
The period at the end, please make it consistent, in some places I find it and it is is missing in some